app/models/form_attachment.rb

```diff
@@ -34,6 +34,22 @@ def get_file
 
   private
 
+  def unlock_pdf(file, file_password)
+    return file unless File.extname(file) == '.pdf'
+
+    pdftk = PdfForms.new(Settings.binaries.pdftk)
+    tmpf = Tempfile.new('decrypted')
+    error_messages = pdftk.call_pdftk(file.tempfile.path, 'input_pw', file_password, 'output', tmpf.path)
+    if error_messages.present?
+      log_message_to_sentry(error_messages, 'warn')
+      raise Common::Exceptions::UnprocessableEntity.new(detail: I18n.t('uploads.pdf.incorrect_password'),
+                                                        source: 'FormAttachment.unlock_pdf')
+    end
+    file.tempfile.unlink
```

Comment:
Unlinking the current tempfile and replacing this with the new one works, but I'd appreciate some extra scrutiny here.